<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextPrmpt</class>
 <widget class="QWidget" name="TextPrmpt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cb_freeze">
     <property name="text">
      <string>Freeze</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="PromptEditor" name="text_prompt"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PromptEditor</class>
   <extends>QTextEdit</extends>
   <header>MJPromptMaker/InheritWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cb_freeze</sender>
   <signal>clicked(bool)</signal>
   <receiver>text_prompt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
